src/components/swap/styleds.tsx

```diff
@@ -86,6 +86,7 @@ const UniswapXShineInner = styled.div`
 
 // overflow hidden to hide the SwapMustacheShadow
 export const SwapOptInSmallContainer = styled.div<{ visible: boolean; shouldAnimate: boolean }>`
+  visibility: ${({ visible }) => (visible ? 'visible' : 'hidden')};
```

Comment:
this change is needed to allow the tests to differentiate between the opt-in mustache being displayed or not (since the component is always rendered so we can't just test for existence)

but the this keeps the component functionally the same:

https://github.com/Uniswap/interface/assets/39385577/1d2323ee-9402-4a10-b4df-05c904557f28